addons/packages/grafana/7.5.11/bundle/config/certificates.lib.yaml

```
@@ -0,0 +1,178 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:template", "template")
+#@ load("@ytt:yaml", "yaml")
+
+#@ def get_issuer():
+apiVersion: cert-manager.io/v1alpha2
+kind: Issuer
+metadata:
+  name: "issuer"
+  namespace: "issuer-namespace"
+spec: {}
+#@ end
+
+#@ def self_signed_issuer():
+spec:
+  #@overlay/match missing_ok=True
+  selfSigned: {}
+#@ end
+
+#@ def ca_issuer():
+spec:
+  #@overlay/match missing_ok=True
+  ca:
+    secretName: "ca-key-pair"
+#@ end
+
+#@ def ca_secret(secret_name):
+spec:
+  ca:
+    secretName: #@ secret_name
+#@ end
+
+#@ def get_self_signed_issuer():
+#@   return overlay.apply(get_issuer(), self_signed_issuer())
+#@ end
+
+#@ def get_ca_issuer():
+#@   return overlay.apply(get_issuer(), ca_issuer())
+#@ end
+
+#@ def generate_self_signed_issuer(name, namespace):
+#@   return overlay.apply(get_self_signed_issuer(), metadata(name, namespace))
+#@ end
+
+#@ def generate_ca_issuer(name, namespace, secret_name):
+#@   return overlay.apply(get_ca_issuer(), metadata(name, namespace), ca_secret(secret_name))
+#@ end
+
+#@ def get_certificate():
+apiVersion: cert-manager.io/v1alpha2
```

```suggestion
apiVersion: cert-manager.io/v1
```

The latest cert-manager versions have dropped `v1alpha2`

---

changed version to v1